### PR TITLE
Assignment

### PR DESCRIPTION
**Pull request by Rantitope**
## What changes are you trying to make? (e.g. Adding or removing code, refactoring existing code, adding reports)
adding codes
## What did you learn from the changes you have made?
creating directories, deleting directories
## Was there another approach you were thinking about making? If so, what approach(es) were you thinking of?
I was thinking of the 1st approach Simeon taught which is simpler
## Were there any challenges? If so, what issue(s) did you face? How did you overcome it?
I had challenges with the app, commiting, pushing etc. Pedram, learning support staff, was of great help in completing the assignment
## How were these changes tested?
I am not sure it was tested. I am open to learning
## Checklist
- [ ] I can confirm that my changes are working as intended
Yes

### CODE REVIEW

File: 02_activities/assignments/assignment.sh [hunk 1]
```diff
@@ -8,17 +8,19 @@ set -x
 # > Add your code below each comment to complete the tasks
 
 # 1. Create 5 directories named dir1, dir2, dir3, dir4, and dir5
-
+mkdir dir1 dir2 dir3 dir4 dir5
 # 2. List the contents of the parent directory to verify the presence of the 5 directories
-
+ls
 # 3. Create 5 text files in dir2 named file1, file2, file3, file4, and file5
+file1.txt file2.txt file3.txt file4.txt file5.txt
```
Comment (dwz92) [inline 02_activities/assignments/assignment.sh:15]: This does not create anything

File: 02_activities/assignments/assignment.sh [hunk 2]
```diff
@@ -8,17 +8,19 @@ set -x
...
 
 # 4. Append the words "Hello world" to dir2/file3
-
+dir2 file3 'Hello world'
```
Comment (dwz92) [inline 02_activities/assignments/assignment.sh:18]: This also doesn't create anything. dir2 is not a command.

File: 02_activities/assignments/assignment.sh [hunk 3]
```diff
@@ -8,17 +8,19 @@ set -x
...
 
 # 4. Append the words "Hello world" to dir2/file3
-
+dir2 file3 'Hello world' 
 # 5. Verify that file3 contains the words "hello world" by printing the contents of the file in the terminal
-
+ls
 # 6. Delete file4
-
+rem file4
```
Comment (dwz92) [inline 02_activities/assignments/assignment.sh:22]: `rm` not rem

Comment (dwz92) [reply 02_activities/assignments/assignment.sh:22]: also missing .txt extension

File: 02_activities/assignments/assignment.sh (hunk 2 above)
Comment (dwz92) [reply 02_activities/assignments/assignment.sh:18]: use `>>`

File: 02_activities/assignments/assignment.sh (hunk 1 above)
Comment (dwz92) [reply 02_activities/assignments/assignment.sh:15]: look into `touch`

**Review by dwz92 — CHANGES_REQUESTED** [review]

**Comment by Rantitope** [comment]
Hello.
Am I expected to re-do the assignment, then submit or take note notes of the errors made?

    On Monday, September 2, 2024 at 02:15:15 PM EDT, Emma ***@***.***> wrote:  
 
 


@dwz92 requested changes on this pull request.

In 02_activities/assignments/assignment.sh:
>  # 3. Create 5 text files in dir2 named file1, file2, file3, file4, and file5
+file1.txt file2.txt file3.txt file4.txt file5.txt

This does not create anything

In 02_activities/assignments/assignment.sh:
>  
 # 4. Append the words "Hello world" to dir2/file3
-
+dir2 file3 'Hello world' 

This also doesn't create anything. dir2 is not a command.

In 02_activities/assignments/assignment.sh:
>  # 6. Delete file4
-
+rem file4

rm not rem

In 02_activities/assignments/assignment.sh:
>  # 3. Create 5 text files in dir2 named file1, file2, file3, file4, and file5
+file1.txt file2.txt file3.txt file4.txt file5.txt

look into touch

In 02_activities/assignments/assignment.sh:
>  
 # 4. Append the words "Hello world" to dir2/file3
-
+dir2 file3 'Hello world' 

use >>

In 02_activities/assignments/assignment.sh:
>  # 6. Delete file4
-
+rem file4

also missing .txt extension and specify dir2

**Comment by dwz92** [comment]
no need to redo, just fix your mistakes and commits again.

But since you closed the PR, remember to create another PR after you fix your mistakes. And send me a text on slack when you are done

**Pull request merged**